.github/workflows/test.yml

```diff
@@ -0,0 +1,28 @@
+name: Test
+
+on:
+  - pull_request
+  - push
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - ruby: "2.4"
+          - ruby: "2.5"
+          - ruby: "2.6"
+          - ruby: "2.7"
```

Comment:
does ruby 3 work as well / can you add that?

Reply:
https://github.com/voxpupuli/librarian/pull/15